index.js

```diff
@@ -30,7 +31,15 @@ Gulp.prototype.watch = function(glob, opt, task) {
     fn = this.parallel(task);
   }
 
-  return vfs.watch(glob, opt, fn);
+  var watcher = chokidar.watch(glob, opt);
+  if (fn) {
```

Comment:
`fn` is the user's optional callback, `opt` is the options object passed through to chokidar. Just like before. Both seem to me to be addressed in the README - you don't think it's adequate enough?